image-layout.md

```diff
@@ -156,4 +161,48 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## index.json
+
+This OPTIONAL file is for use indexing references and elements of the image-layout, where walking the `refs` and `blobs` is sub-optimal.
```

Comment:
> updated.

You're still referencing `blobs` in this line (which is what [started][1] the sub-thread on this line).

[1]: https://github.com/opencontainers/image-spec/pull/533/files#r98119434